bin/docker/kafka_bridge_run.sh

```diff
@@ -69,5 +66,7 @@ if [ "$FIPS_MODE" = "disabled" ]; then
     export JAVA_OPTS="${JAVA_OPTS} -Dcom.redhat.fips=false"
 fi
 
+for var in ${!KAFKA_BRIDGE_*} ; do unset ${var} ; done
```

Comment:
let's add a comment for now, something like ...

"stripping these env vars out because Quarkus would handle them as configuration parameters"